docs/_source/docs/stack_config.rst

```diff
@@ -80,6 +80,13 @@ if a Stack fetches an output value from another Stack using the
 ``stack_output`` resolver, that Stack is automatically added as a dependency,
 and that Stack need not be added as an explicit dependency.
 
+.. warning::
+   Be careful about how you structure dependencies. It is possible to create circular
+   dependencies accidentally, where multiple stacks depend on each other. Sceptre
+   will detect this and raise an error, blocking this sort of setup. This will *always*
+   be the case when you set dependencies on a StackGroup Config referencing stacks
+   inside that StackGroup.
```

Comment:
I think this description is a bit confusing and misleading because I don't think that applies to every case.

Given the stest.zip sample.  if i set any dependency in the root StackConfig (config/config.yaml) it will cause a cycle error.  if I set a dependency in a config one level down (config/nba/config.yaml) it works fine.   I believe both configs __NOT__  `referencing stacks inside that StackGroup`

Reply:
I'll try clarifying this. I think I see where your confusion comes from.